src/Raven.Client/Documents/AI/AiOperations.cs

```diff
@@ -64,121 +64,41 @@ public AiOperations ForDatabase(string databaseName)
-    /// <param name="identifier">The identifier of the AI agent to chat with.</param>
-    /// <param name="prompt">The initial user prompt.</param>
-    /// <param name="builder">A builder function to define RAG parameters.</param>
-    /// <returns>The result of the chat.</returns>
-    public ChatResult<TSchema> StartChat<TSchema>(string identifier, string prompt, Func<AiAgentParametersBuilder, AiAgentParametersBuilder> builder) where TSchema : new()
-    {
-        return AsyncHelpers.RunSync(() => StartChatAsync<TSchema>(identifier, prompt, builder));
-    }
+        public AiAgentParametersBuilder AddParameter(string key, string value)
+        {
+            _parameters[key] = value;
+            return this;
+        }
 
-    /// <summary>
-    /// Asynchronously starts a new chat with an AI agent using a dictionary of parameters.
-    /// </summary>
-    /// <typeparam name="TSchema">The schema type for the chat.</typeparam>
-    /// <param name="identifier">The identifier of the AI agent to chat with.</param>
-    /// <param name="prompt">The initial user prompt.</param>
-    /// <param name="parameters">Optional dictionary of chat parameters.</param>
-    /// <returns>The result of the chat.</returns>
-    public async Task<ChatResult<TSchema>> StartChatAsync<TSchema>(string identifier, string prompt, Dictionary<string, object> parameters = null, CancellationToken token = default) where TSchema : new()
-    {
-        return await _executor.SendAsync(new StartChatOperation<TSchema>(identifier, prompt, parameters), token).ConfigureAwait(false);
+        public Dictionary<string, object> GetParameters() => _parameters.Count == 0 ? null : _parameters;
     }
-
+    
     /// <summary>
-    /// Synchronously starts a new chat with an AI agent using a dictionary of parameters.
+    /// Create starts a new chat with an AI agent using a dictionary of parameters.
     /// </summary>
-    /// <typeparam name="TSchema">The schema type for the chat.</typeparam>
-    /// <param name="identifier">The identifier of the AI agent to chat with.</param>
-    /// <param name="prompt">The initial user prompt.</param>
-    /// <param name="parameters">Optional dictionary of chat parameters.</param>
-    /// <returns>The result of the chat.</returns>
-    public ChatResult<TSchema> StartChat<TSchema>(string identifier, string prompt, Dictionary<string, object> parameters = null) where TSchema : new()
-    {
-        return AsyncHelpers.RunSync(() => StartChatAsync<TSchema>(identifier, prompt, parameters));
-    }
-
+    /// <typeparam name="TSchema">The schema type for the chat response.</typeparam>
+    /// <param name="agentName">The name of the AI agent to chat with.</param>
+    /// <param name="parameters">Required chat parameters.</param>
+    public IChatOperations<TSchema> CreateChat<TSchema>(string agent, Dictionary<string, object> parameters = null) where TSchema : new() => new Chat<TSchema>(this, agent, parameters);
```

Comment:
in XML y ou have agentName, but parameter is called agent. Also I do not know what you want to put there? the agentId probably?